misc/dist/html/full-size.html

```diff
@@ -52,6 +52,20 @@
 	margin: auto;
 }
 
+#status-splash.show-image_false {
```

Comment:
@elpozewaunig Before you merge your 2 commits together, I would just replace the `_` with `--`, but it's just a personal preference.

```suggestion
#status-splash.show-image--false {
```

Reply:
Done :+1: